proto.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016-2020 Fraunhofer AISEC
```

Comment:
```suggestion
 * Copyright 2021-2022 Fraunhofer AISEC
```

Reply:
I think we started with Go last year? :)

Reply:
The overall project started in 2016, so I would use this as a start year

Reply:
```suggestion
 * Copyright 2016-2022 Fraunhofer AISEC
```

Reply:
I changed it, accordingly.